trade-finance-manager-api/src/v1/helpers/amendment.helpers.test.js

```diff
@@ -1036,12 +993,8 @@ describe('internalAmendmentEmail()', () => {
   it('Should call the expected function with expected arguments set', async () => {
     await internalAmendmentEmail('1234567890');
 
-    expect(sendEmailApiSpy).toHaveBeenCalledWith(
-      CONSTANTS.EMAIL_TEMPLATE_IDS.INTERNAL_AMENDMENT_NOTIFICATION,
-      expect.any(String),
-      {
-        ukefFacilityId: '1234567890',
-      },
-    );
+    expect(sendEmailApiSpy).toHaveBeenCalledWith(CONSTANTS.EMAIL_TEMPLATE_IDS.INTERNAL_AMENDMENT_NOTIFICATION, expect.any(String), {
```

Comment:
```suggestion    expect(sendEmailApiSpy).toHaveBeenCalledWith(CONSTANTS.EMAIL_TEMPLATE_IDS.INTERNAL_AMENDMENT_NOTIFICATION, process.env.THE_VARIABLE_CALLED_WTIH, {
```
This might fix test